.github/workflows/dotnet.yml

```
@@ -0,0 +1,93 @@
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+          
+    - name: Restore and bundle
+      run: |
+        # Find and restore solution
+        solution=$(find . -name "*.sln" -type f)
+        dotnet restore "$solution"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)